compute_endpoint/tests/integration/endpoint/endpoint/test_endpoint_manager.py

```diff
@@ -59,46 +59,46 @@ def test_double_configure(self):
         with pytest.raises(Exception, match="ConfigExists"):
             manager.configure_endpoint(config_dir, None)
 
-    @pytest.mark.parametrize("mt", [None, True, False])
-    def test_configure_multi_tenant_existing_config(self, mt):
+    @pytest.mark.parametrize("mu", [None, True, False])
+    def test_configure_multi_user_existing_config(self, mu):
```

Comment:
These tests are what I was referring to -- it may be beneficial to add a truthy value to these parameters.

Reply:
Resolving for the same reason as above